docs/pages/enterprise/workflow/ssh-approval-pagerduty.mdx

```diff
@@ -82,10 +85,22 @@ Run `./install` from `teleport-pagerduty` or place the executable in the appropr
 
 Teleport's PagerDuty plugin has its own configuration file in TOML format. Before starting the plugin for the first time, you'll need to generate and edit that config file.
 
+<Tabs>
+<TabItem label="Server">
 ```code
 $ teleport-pagerduty configure > teleport-pagerduty.toml
 $ sudo mv teleport-pagerduty.toml /etc
 ```
+</TabItem>
+<TabItem label="Docker">
+```code
+$ docker run -it quay.io/gravitational/teleport-plugin-pagerduty:(=teleport.plugin.version=) configure > teleport-pagerduty.toml
+$ mkdir config
+$ cp teleport-pagerduty.toml config
```

Comment:
Since the "Server" tab uses `sudo mv` to move the file, should the "Docker" tab include this as well? Same for the `mkdir` command in the "Docker" tab.